api/base/exceptions.py

```diff
@@ -41,3 +41,8 @@ def json_api_exception_handler(exc, context):
 class Gone(APIException):
     status_code = status.HTTP_410_GONE
     default_detail = ('The requested resource is no longer available.')
+
+
+class InvalidFilter(ParseError):
```

Comment:
Note that `NotAuthenticated` and `AuthenticationFailed` both result in a `401 UNAUTHORIZED` response.

That said, i don't feel strongly about the name I chose; I'll go ahead and add `Error`.